composer.lock

```diff
@@ -3186,23 +3296,26 @@
         },
         {
             "name": "codeception/phpunit-wrapper",
-            "version": "7.7.1",
+            "version": "6.6.1",
```

Comment:
Can you explain why you have changed all the composer.lock ? Especially the update of dependencies version ?
For example here, why have you downgraded the codeception/phpunit-wrapper required version ?

Reply:
I update the composer in the wrong way. The commit c6647cf0f68203874be71030727ed932ae8384da correct this issue.